### PR TITLE
Uitdatabank/search-api-guides-pagination

### PR DESCRIPTION
**Pull request by corneelwille**
### Added
- documentation page for pagination
- add link to pagination docs to sidebar

### CODE REVIEW

File: projects/uitdatabank/docs/search-api/pagination.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Pagination
+
+Every search endpoint supports pagination using two url parameters:
+-   `start`: the number of results to skip (`int`, defaults to 0)
+-   `limit`: the number of results to return in a single page (`int`, defaults to 30)
+
+Pagination will only work for a limited result set:
+-   The maximum value for for `start` is set to 10.000
```
Comment (bertramakers) [inline projects/uitdatabank/docs/search-api/pagination.md:8]:
```suggestion
-   The maximum value for for `start` is set to `10000`
```

File: projects/uitdatabank/docs/search-api/pagination.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+-   The maximum value for `limit` is set to 2.000
```
Comment (bertramakers) [inline projects/uitdatabank/docs/search-api/pagination.md:9]:
```suggestion
-   The maximum value for `limit` is set to `2000`
```

File: projects/uitdatabank/docs/search-api/pagination.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+<!-- theme: info -->
+
+> The response also includes the limit included in the request, as `itemsPerPage`. This is because the API might decide to return fewer results if the specified limit is too high. Because of this, you should never assume that the specified limit will also be respected and you should always check `itemsPerPage`.
```
Comment (bertramakers) [inline projects/uitdatabank/docs/search-api/pagination.md:13]: I also read this in the old documentation, but it's actually not true (anymore). We now just return an error if a limit is given that is too high. So the `itemsPerPage` is always the same as the given `limit` (or the default value if none is given)

File: projects/uitdatabank/docs/search-api/pagination.md (hunk 1 above)
Comment (corneelwille) [reply projects/uitdatabank/docs/search-api/pagination.md:8]: Useful insight, thanks! 👍

**Review by bertramakers — APPROVED** [review]

**Comment by bertramakers** [comment]
@corneelwille I've resolved the merge conflict in this PR. It was in the sidebar config. The "default filters" and "sorting" links were added in the same place in 2 separate PRs, so git didn't know what to do with it. I've put the "default filters" link above "pagination" and "sorting" for now, but feel free to move it.

**Pull request merged**